src/api/python/setup.py.in

```diff
@@ -0,0 +1,21 @@
+from distutils.core import setup
```

Comment:
Add a header and documentation here.

Reply:
Added with d036f2664a5fad25209e55475242dc2df7001325. Let me know if that's not sufficient.